### ISSUE TITLE
Is it a good phenomenon that copy occupies 100%? 

### ISSUE THREAD

**Issue by wjd000** [develsoftware/GMinerRelease#234]
I don't know much about GPU. I found that copy occupies 100% while using NVIDIA RTX 30, but this is not the case with other mining software. What is this? Is this a good phenomenon?

**Comment by igoralebar** [comment]
why does it bother you?
is something not performing up to your expectations?

**Comment by wjd000** [comment]
No, everything is fine, it's just that other mining software doesn't have this situation. The efficiency of gminer is higher than other software I have used. So I’m just curious if it’s because gminer uses copy and other software doesn’t.

**Comment by igoralebar** [comment]
are you monitoring GPU with windows task manager or something?
and what does copy graph tells you exactly about performance? how do you imagine graph drawn in task manager, that says Copy, is translated into gminer performance?

**Comment by wjd000** [comment]
> 
> 
> are you monitoring GPU with windows task manager or something?
> and what does copy graph tells you exactly about performance? how do you imagine graph drawn in task manager, that says Copy, is translated into gminer performance?

Other mining software occupies only 1% of Copy, but gminer occupies 100%, so I am just wondering whether it is because gminer uses Copy to make it more efficient.
